tests/test_ssm.py

```diff
@@ -47,3 +47,60 @@ def test_ssm_parameter_not_secure(self):
         resources = p.run()
         self.assertEqual(len(resources), 1)
         self.addCleanup(client.delete_parameters, Names=['test-name', 'secure-test-name'])
+
+    @functional
+    def test_ssm_activation_expired(self):
+        session_factory = self.replay_flight_data("test_ssm_activation_expired")
+        client = session_factory().client("ssm")
+        p = self.load_policy(
+            {
+                "name": "ssm-list-expired-activations",
+                "resource": "ssm-activation",
+                "filters": [{
+                  "type": "value",
+                  "key": "Expired",
+                  "value": True
+                }]
+            },
+            session_factory=session_factory,
+        )
+        resources = p.run()
+        self.assertEqual(len(resources), 2)
+
+    @functional
+    def test_ssm_activation_expired_no_registrations(self):
+        session_factory = self.replay_flight_data("test_ssm_activation_expired_no_registrations")
+        client = session_factory().client("ssm")
+        p = self.load_policy(
+            {
+                "name": "ssm-list-expired-activations-no-registrations",
+                "resource": "ssm-activation",
+                "filters": [{
+                  "type": "value",
+                  "key": "Expired",
+                  "value": True
+                }, {
+                  "type": "value",
+                  "key": "RegistrationsCount",
+                  "value": 0
+                }]
+            },
+            session_factory=session_factory,
+        )
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
+    
+    @functional
+    def test_ssm_get_manager_instances(self):
+        session_factory = self.replay_flight_data("test_ssm_get_managed_instances")
+        client = session_factory().client("ssm")
+        p = self.load_policy(
+            {
+                "name": "ssm-get-managed-instances",
+                "resource": "ssm-managed-instances"
+            },
+            session_factory=session_factory,
+        )
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
```

Comment:
please assert an actual value/attribute of the managed instance (instance-id) to verify that the test actual received a value that it expected.